ast/src/parsed/display.rs

```diff
@@ -598,7 +598,7 @@ fn format_list<L: IntoIterator<Item = I>, I: Display>(list: L) -> String {
 impl<E: Display> Expression<E> {
     pub fn precedence(&self) -> Option<ExpressionPrecedence> {
         match self {
-            Expression::UnaryOperation(op, _) => Some(op.precedence()),
+            Expression::UnaryOperation(UnaryOperation { op, expr: _ }) => Some(op.precedence()),
             Expression::BinaryOperation(_, op, _) => Some(op.precedence()),
             _ => None,
         }
```

Comment:
I think we should now change the function `format_unary_operation` to just be the Display impl on UnaryOperation.

Reply:
Same situation as with BinaryExpression.